configure.ac

```diff
@@ -397,24 +398,25 @@ AC_MSG_NOTICE([Building documents only])
 ],
 [
 AC_MSG_NOTICE([UCX build configuration:])
-AC_MSG_NOTICE([      Build prefix:   ${prefix}])
-AC_MSG_NOTICE([ Configuration dir:   ${ucx_conf_dir}])
-AC_MSG_NOTICE([Preprocessor flags:   ${BASE_CPPFLAGS}])
-AC_MSG_NOTICE([        C compiler:   ${CC} ${BASE_CFLAGS}])
-AC_MSG_NOTICE([      C++ compiler:   ${CXX} ${BASE_CXXFLAGS}])
-AC_MSG_NOTICE([      Multi-thread:   ${mt_enable}])
-AC_MSG_NOTICE([      NUMA support:   ${numa_enable}])
-AC_MSG_NOTICE([         MPI tests:   ${mpi_enable}])
-AC_MSG_NOTICE([       VFS support:   ${vfs_enable}])
-AC_MSG_NOTICE([     Devel headers:   ${enable_devel_headers}])
-AC_MSG_NOTICE([          Bindings:   <$(echo ${build_bindings}|tr ':' ' ') >])
-AC_MSG_NOTICE([       UCS modules:   <$(echo ${ucs_modules}|tr ':' ' ') >])
-AC_MSG_NOTICE([       UCT modules:   <$(echo ${uct_modules}|tr ':' ' ') >])
-AC_MSG_NOTICE([      CUDA modules:   <$(echo ${uct_cuda_modules}|tr ':' ' ') >])
-AC_MSG_NOTICE([      ROCM modules:   <$(echo ${uct_rocm_modules}|tr ':' ' ') >])
-AC_MSG_NOTICE([        IB modules:   <$(echo ${uct_ib_modules}|tr ':' ' ') >])
-AC_MSG_NOTICE([       UCM modules:   <$(echo ${ucm_modules}|tr ':' ' ') >])
-AC_MSG_NOTICE([      Perf modules:   <$(echo ${ucx_perftest_modules}|tr ':' ' ') >])
+AC_MSG_NOTICE([         Build prefix  :   ${prefix}])
+AC_MSG_NOTICE([    Configuration dir  :   ${ucx_conf_dir}])
+AC_MSG_NOTICE([   Preprocessor flags  :   ${BASE_CPPFLAGS}])
+AC_MSG_NOTICE([           C compiler  :   ${CC} ${BASE_CFLAGS}])
+AC_MSG_NOTICE([         C++ compiler  :   ${CXX} ${BASE_CXXFLAGS}])
+AC_MSG_NOTICE([         Multi-thread  :   ${mt_enable}])
+AC_MSG_NOTICE([         NUMA support  :   ${numa_enable}])
+AC_MSG_NOTICE([            MPI tests  :   ${mpi_enable}])
+AC_MSG_NOTICE([          VFS support  :   ${vfs_enable}])
+AC_MSG_NOTICE([        Devel headers  :   ${enable_devel_headers}])
+AC_MSG_NOTICE([CUDA support in io_demo:   ${with_iodemo_cuda}])
```

Comment:
1. pls move it to end of list
2. why added space before ':' for other items?
3. can make it shorted: "io_demo CUDA support: "

Reply:
1 - done, thanks.
2,3 - actually initially I put it at the end of the list and called it `io_demo CUDA support`. Then I decided to change, because:
- A sentence starting with a lowercase looked weird to me.
- Last elements in the output are lists, putting a single param after it makes it look weird:
```
configure: =========================================================
configure: UCX build configuration:
configure:            Build prefix:   /labhome/lgenkin/install
configure:       Configuration dir:   ${prefix}/etc/ucx
configure:      Preprocessor flags:   -DCPU_FLAGS="" -I${abs_top_srcdir}/src -I${abs_top_builddir} -I${abs_top_builddir}/src
configure:              C compiler:   gcc -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wno-pointer-sign -Werror-implicit-function-declaration -Wno-format-zero-length -Wnested-externs -Wshadow -Werror=declaration-after-statement
configure:            C++ compiler:   g++ -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch
configure:            Multi-thread:   enabled
configure:            NUMA support:   enabled
configure:               MPI tests:   disabled
configure:             VFS support:   yes
configure:           Devel headers:   no
configure:                Bindings:   < >
configure:             UCS modules:   < fuse >
configure:             UCT modules:   < cuda ib rdmacm cma knem >
configure:            CUDA modules:   < gdrcopy >
configure:            ROCM modules:   < >
configure:              IB modules:   < >
configure:             UCM modules:   < cuda >
configure:            Perf modules:   < cuda >
configure:    io_demo CUDA support:   no
configure: =========================================================
```

WDYT?

Reply:
Ok, let's move back to after devel-headers. IMO can keep as lowercase since it's the exact test name

Reply:
Done